### PR TITLE
Fix #4260 - bug in interaction between rownumberpaging and include

### LINKED ISSUE #4260: SQL2008 query bug with Include and skip
Changes made in 804458aa576723d8732f65605effde5b7fb7145a cause IncludeSqlServerTest.Include_with_skip to start to fail on SQL Server 2008:

```
System.InvalidOperationException : Sequence contains no matching element
   at System.Linq.Enumerable.Last[TSource](IEnumerable`1 source, Func`2 predicate)
   at Microsoft.Data.Entity.Query.ExpressionVisitors.Internal.IncludeExpressionVisitor.
```

Full stack trace on our CI server

cc @anpete

### PR DESCRIPTION
**Pull request by natemcmaster**
cc @anpete

### CODE REVIEW

File: src/EntityFramework.Relational/Query/ExpressionVisitors/Internal/IncludeExpressionVisitor.cs [hunk 1]
```diff
@@ -207,7 +207,11 @@ yield return
                 else
                 {
                     var principalTable
-                        = selectExpression.Tables.Last(t => t.QuerySource == querySource);
+                        = (selectExpression.Tables.Count == 1)
```
Comment (anpete) [inline src/EntityFramework.Relational/Query/ExpressionVisitors/Internal/IncludeExpressionVisitor.cs]: Can we use SelectExpression.GetTableForQuerySource instead?

Comment (natemcmaster) [inline src/EntityFramework.Relational/Query/ExpressionVisitors/Internal/IncludeExpressionVisitor.cs]: Nope.

Comment (natemcmaster) [inline src/EntityFramework.Relational/Query/ExpressionVisitors/Internal/IncludeExpressionVisitor.cs]: Tried, but this method gets `FirstOrDefault` and we need `Last()`.

**Comment by anpete** [comment]
:shipit:

**Pull request merged**